pkg/controller/restapi/operation/vdri/models.go

```diff
@@ -0,0 +1,37 @@
+/*
+ *
+ * Copyright SecureKey Technologies Inc. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * /
+ *
+ */
+
+package vdri
+
+import (
+	vdricommand "github.com/hyperledger/aries-framework-go/pkg/controller/command/vdri"
+	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
+)
+
+// createPublicDIDRequest model
+//
+// This is used for operation to create public DID
+//
+// swagger:parameters createPublicDID
+type createPublicDIDRequest struct { // nolint: unused,deadcode
+	// Params for creating public DID
+	//
+	// in: path
+	*vdricommand.CreatePublicDIDArgs
+}
+
+// createPublicDIDResponse model
+//
+// This is used for returning public DID created
+//
+// swagger:response createPublicDIDResponse
+type createPublicDIDResponse struct {
```

Comment:
Why is this struct used?

Reply:
it is used in unit tests, so didn't require that tag.